src/TestContext.elm

```diff
@@ -11,16 +14,20 @@ module TestContext
         , done
         , expectLastEffect
         , expectModel
+        , expectPageChange
         , expectView
         , expectViewHas
         , fail
+        , fillIn
+        , fillInTextarea
         , routeChange
         , shouldHave
         , shouldHaveLastEffect
         , shouldHaveView
         , shouldNotHave
         , simulate
         , update
+        , within
         )
```

Comment:
FYI, I opened a [PR](https://github.com/NoRedInk/NoRedInk/pull/19554) last night that took functions like `clickOn` and `ifThenElse` from `ui/tests/Page/Teach/Assignment/Form/UiActions.elm` and put them in a new file `ui/tests/SpecHelpers/UiActions.elm`. However, I think putting the helper functions in this file makes perfect sense, so once your PR is merged, I can move the `UiActions` functions into this file, if that sounds OK to you?